StyleGuide.md

```diff
@@ -831,6 +832,34 @@ If a feature has been backported to a previous version in a patch release, use t
 
 Version tags should only refer to the version of the docset that contains them. For example, the version tag `{% include_cached new-in.html version="v21.1.9" %}` should only be on pages in `v21.1` directories.
 
+### Version references
+
+To refer to a static version of CockroachDB:
+
+~~~
+{{site.versions["v22.2"]}}
+~~~
+
+To dynamically refer to the stable version of CockroachDB, as determined each time the site is built:
+
+~~~
+{{site.versions["stable"]}}
+~~~
+
+**Warning**: If you use a `stable` link on a versioned page which is for a previous version, the link points to a different version  of CockroachDB than the version the page documents. Similarly, if you use a `stable` link on a page for the current version and then a new version is added, the link points to a different version than the version the page documents. If this is a problem, use one of the following methods instead.
+
+To refer to a page's minor version (for example, {{page.version.version}}), which matches its top-level subdirectory within the docs repo:
```

Comment:
Don't you want a real example of a minor version?  The version should be in backticks.

Reply:
Fixed (we synced about backticks, not needed)